sdk/python/kfp/cli/pipeline.py

```diff
@@ -30,16 +30,21 @@ def pipeline():
     "--pipeline-name",
     help="Name of the pipeline."
 )
+@click.option(
+    "-d",
+    "--pipeline-description",
+    help="Description for the pipeline."
+)
 @click.argument("package-file")
 @click.pass_context
-def upload(ctx, pipeline_name, package_file):
+def upload(ctx, package_file, pipeline_name, pipeline_description=None):
```

Comment:
Is there any reason for changing the order of `pipeline_name` and `package_file`, suggest we keep the old to not break the previous behaviour. 

nit: suggest to keep the name to `description` to be aligned with the arguments for `client.upload_pipeline`, [docs](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html#kfp.Client.upload_pipeline)

Reply:
@NikeNano Made the changes based on your suggestions. The order of arguments is changed to original and renamed the `pipeline_description` to just `description`.

Reply:
Realised it would not have brooken anything since this uses click and then the order should not matter.